### PR TITLE
fix: fiixed PR conflicts

### PR DESCRIPTION
**Pull request by 23po**
# Description

This PR is a better fix to conflicts from my first PR 

-- including the package.json.lock file,
-- and had dead code in variables.scss,

Other problems raised in the first review are also addressed, the html selector is also renamed to elewa-group-team-member from elewa-group-elewa-group-team-member. 

Refer to my first PR here: [https://github.com/italanta/elewa-group/pull/86#issue-1577927358]

# Solution

Created a Reusable Single team member feature. When a user clicks on a specific member of the team from the team member carousel, the single page view of that team member should open.

The change does not require any new/special dependencies

Adds #16

To achieve this:

- The change includes a new banners-features-components module 
- A new Elewa-Group-Single-Page-View Component and dependent files (scss, html, typescript files) and configuration
- The Module is integrated with the App module, while component renders via the Home component
- Finally styling and logic for the

Fixes # (issue) - *Example*: Fixes # ([S4Y-400](https://www.w3.org/Provider/Style/dummy.html), [S4Y-100](https://www.w3.org/Provider/Style/dummy.html))

## Type of change
- [x] Bug fix (non-breaking change which fixes an issue)

# Screenshots (Desktop/Mobile)
217943837-4ebc0647-2252-48d8-8646-99dfd4a57c9f
217944145-c35dbe29-9382-41c1-821f-8a2e9addc034

# Video 
[screen-capture.webm](https://user-images.githubusercontent.com/48943229/218410879-a06b27d3-d502-47e2-9145-02423f02c7d8.webm)
(Desktop/Mobile)

# How Has This Been Tested?

The feature renders on localhost 4200. The component is added as part of the home.page.component.html

The header component may run into the top of this component, so it is disabled to fulfill the look of the designs shared for this component. 

# Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by IanOdhiambo9 — COMMENTED** [review]
Good job,
However, I need to confirm the carousel because there's no video provided.

**Comment by 23po** [comment]
okay, however I added a new video section on my PR and pasted a video link --> [screen-capture.webm](https://user-images.githubusercontent.com/48943229/218410879-a06b27d3-d502-47e2-9145-02423f02c7d8.webm)

**Review by IanOdhiambo9 — APPROVED** [review]
Great Job

I've aproved but please resolve conflicts first so I can merge

File: apps/elewa-group-website/src/app/app.component.html [hunk 1]
```diff
@@ -1 +1,2 @@
-<router-outlet></router-outlet>
\ No newline at end of file
+<elewa-group-home-page></elewa-group-home-page>
```
Comment (IanOdhiambo9) [inline apps/elewa-group-website/src/app/app.component.html:1]: remove this

**Pull request merged**